### PR TITLE
Added the openBookingWindow to the frontpage

### PR DESCRIPTION
**Pull request by alfredbrannare**
Added the openBookingWindow to the frontpage card buttons.

https://github.com/user-attachments/assets/28068c1a-dfda-4138-a81d-9527d222f37f

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -44,6 +44,17 @@ async function highestRanking() {
               card.querySelector(".bookBtn").textContent = "Take challenge online";
               card.querySelector(".typeRoom").textContent = "";
             }
```
Comment (Emilyempa) [inline main.js:46]: Nice touch to be able to open booking page from our main

**Review by Emilyempa — APPROVED** [review]
Fine with me to merge. Nice job!

**Review by Paajt — APPROVED** [review]
Looks great! Ok to merge! 👍

**Comment by NordicNomadicLife** [comment]
Looks good 👍

**Pull request merged**